R/zzz.R

```diff
@@ -0,0 +1,20 @@
+# Using this approach from the cli package on startup messages.
+
+## nocov start
+
+.onLoad <- function(libname, pkgname) {
+    packageStartupMessage(cli_inform(enw_startup_message()))
+}
+
+enw_startup_message <- function(){
+    cache_location <- Sys.getenv("enw_cache_location")
+    if (check_environment_setting(cache_location)) {
+        msg <- "`enw_cache_location` is not set it. Set it using `enw_set_cache` to reduce future Stan compilation times"
```

Comment:
should we suggest using the R cache here as I think most users won't know about it?